### ISSUE TITLE
Get Request with Data

### ISSUE THREAD

**Issue by rag1998** [bulkan/robotframework-requests#148]
We recently faced a need for doing Get Requests that can have a body passed to them, we were able to implement this using python requests library for now, by doing:

```Python
def get_with_body(url, headers, data):
    headers = {
        'Content-Type': 'application/json'
    }
    data = {"latitude":"30.496346","longitude":"-87.640356"}
    r = requests.get(url, headers=headers, json=data)
```

It would be great if you can extend your library to include this functionality.

**Comment by vkosuri** [comment]
@rag1998 thank will add this feature soon.

**Comment by rag1998** [comment]
Thanks :)

**Comment by vkosuri** [comment]
Difference between ``data`` and ``json`` parameters

``` Python
import requests
import json
d = {'a': 1}
response = requests.get(url, json=d)
```

The ``json`` parameter does indeed set the ``content-type`` in the headers to ``application/json``.

``` Python
import requests
import json
d = {'a': 1}
response = requests.get(url, data=json.dumps(d))
```

The ``data`` parameter, the ``content-type`` in the headers would need to be set manually.

**Comment by vkosuri** [comment]
@rag1998 this issue was addressed with #154
